sycl/plugins/level_zero/CMakeLists.txt

```diff
@@ -93,39 +75,39 @@ add_sycl_plugin(level_zero
-    "../unified_runtime/ur/adapters/level_zero/kernel.cpp"
-    "../unified_runtime/ur/adapters/level_zero/memory.cpp"
-    "../unified_runtime/ur/adapters/level_zero/platform.cpp"
-    "../unified_runtime/ur/adapters/level_zero/program.cpp"
-    "../unified_runtime/ur/adapters/level_zero/queue.cpp"
-    "../unified_runtime/ur/adapters/level_zero/sampler.cpp"
-    "../unified_runtime/ur/adapters/level_zero/usm.cpp"
-    "../unified_runtime/ur/adapters/level_zero/usm_p2p.cpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/ur/ur.hpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/ur/ur.cpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/ur_level_zero.hpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/command_buffer.hpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/adapter.hpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/common.hpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/context.hpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/device.hpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/event.hpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/image.hpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/kernel.hpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/memory.hpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/platform.hpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/program.hpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/queue.hpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/sampler.hpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/usm.hpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/ur_level_zero.cpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/adapter.cpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/command_buffer.cpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/common.cpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/context.cpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/device.cpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/event.cpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/image.cpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/kernel.cpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/memory.cpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/platform.cpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/program.cpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/queue.cpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/sampler.cpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/usm.cpp"
+    "${UNIFIED_RUNTIME_SOURCE_DIR}/source/adapters/level_zero/usm_p2p.cpp"
```

Comment:
Ideally I think we should get this list of source files from UR, either in a variable we define or by querying the list of source files from the L0 adapter target. That way we won't need to update this list if/when a new file is added to the L0 target in UR.